js/src/lib/components/InferenceWidget/shared/WidgetTextarea/WidgetTextarea.svelte

```diff
@@ -110,20 +73,21 @@
 
 <WidgetLabel {label}>
 	<svelte:fragment slot="after">
+		<!-- `whitespace-pre-wrap inline-block` are needed to get correct newlines from `el.textContent` on Chrome -->
 		<span
 			class="{isLoading ? 'pointer-events-none' : ''} {label
 				? 'mt-1.5'
 				: ''} block overflow-auto resize-y py-2 px-3 w-full {size === 'small'
 				? 'min-h-[42px]'
-				: 'min-h-[144px]'} max-h-[500px] whitespace-pre-wrap border border-gray-200 rounded-lg shadow-inner outline-none focus:ring focus:ring-blue-200 focus:shadow-inner dark:bg-gray-925"
```

Comment:
added this accidental omission of `border` back in https://github.com/huggingface/hub-docs/commit/d2c0de490c2be461569a3393c0f93c030f5c51bf